.github/workflows/build_documentation.yml

```diff
@@ -14,6 +14,6 @@ jobs:
       package: course
       path_to_docs: course/chapters/
       additional_args: --not_python_module
-      languages: en es fr he pt th tr
+      languages: en es fr he ko pt th tr
```

Comment:
FYI I've added Korean to the list here so we can build the docs on the website once this PR is merged :)